HomeHealthServices/R4/resources/Questionnaire-R4-HomeHealthServicesFaceToFace.json

```diff
@@ -0,0 +1,1219 @@
+                  "valueCoding": {
+                    "display": "%"
+                  }
+                }
+              ],
+              "type": "decimal"
+            },
+            {
+              "linkId": "7.1.3",
+              "text": "O2 Sat with supplemental oxygen",
+              "type": "group",
+              "extension": [
+                {
+                  "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+                  "valueCodeableConcept": {
+                    "coding": [
+                      {
+                        "system": "http://hl7.org/fhir/questionnaire-item-control",
+                        "code": "gtable"
+                      }
+                    ]
+                  }
+                }
+              ],
+              "item": [
+                {
+                  "linkId": "7.1.3.1",
+                  "text": "O2 Sat",
+                  "extension": [
+                    {
+                      "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-unit",
+                      "valueCoding": {
+                        "display": "%"
+                      }
+                    }
+                  ],
+                  "type": "decimal"
+                },
+                {
+                  "linkId": "7.1.3.2",
+                  "text": "O2 flow rate (LPM)",
+                  "extension": [
+                    {
+                      "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-unit",
+                      "valueCoding": {
+                        "display": "lpm"
+                      }
+                    }
+                  ],
+                  "type": "decimal"
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire",
+              "valueCanonical": "questionnaire/physical-exam"
+            }
+          ],
+          "linkId": "7.2",
+          "type": "display",
+          "text": "This is a placeholder for Physical Examination"
+        }
+      ]
+    },
+    {
+      "linkId": "8",
+      "type": "group",
+      "text": "Confined to Home Assessment",
+      "item": [
+        {
+          "type": "group",
+          "linkId": "8.1",
+          "text": "Does beneficiary, because of illness or injury, need: (check all that apply)",
+          "item": [
+            {
+              "type": "boolean",
+              "linkId": "8.1.1",
+              "text": "supportive devices"
+            },
+            {
+              "type": "boolean",
+              "linkId": "8.1.2",
+              "text": "special transportation"
+            },
+            {
+              "type": "boolean",
+              "linkId": "8.1.3",
+              "text": "assistance to leave residence"
+            },
+            {
+              "type": "string",
```

Comment:
this should be "text" to be consistent with other elements and so there is sufficient space to fully describe problem